security/pkg/pki/util/crypto.go

```diff
@@ -49,10 +49,11 @@ func ParsePemEncodedCertificate(certBytes []byte) (*x509.Certificate, error) {
 
 // ParsePemEncodedCertificateChain constructs a slice of `x509.Certificate`
 // objects using the given a PEM-encoded certificate chain.
```

Comment:
please update the comments

Reply:
Hi @hzxuzhonghu @lei-tang , this comments is addressed, PTAL.